src/viam/components/camera/camera.py

```diff
@@ -48,6 +49,21 @@ async def get_image(self, mime_type: str = "", *, timeout: Optional[float] = Non
         """
         ...
 
+    @abc.abstractmethod
+    async def get_images(self, *, timeout: Optional[float] = None, **kwargs) -> Tuple[List[Union[Image, RawImage]], datetime]:
+        """Get simultaneous images from different sensors, along with associated metadata (just timestamp for now).
+        It's not for getting a time series of images from the same sensor.
+
+        Returns:
+            Tuple[List[Union[Image, RawImage]], datetime]:
+                - List[Union[Image, RawImage]]:
+                  The list of images returned from the camera system.
```

Comment:
This is missing the "source_name" associated with the image, which is a string stating the sensor's name that produced the image